src/registrar/migrations/0071_alter_contact_first_name_alter_contact_last_name_and_more.py

```diff
@@ -5,7 +5,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
I think this change was made by mistake, revert it if possible

Reply:
linter would not pass without it

Reply:
Zander figured this out.  May be related to an updated linter when I added tblib and rebuilt.